contracts/governance/Vesting/fouryear/FourYearVestingFactory.sol

```diff
@@ -26,6 +26,7 @@ contract FourYearVestingFactory is IFourYearVestingFactory, Ownable {
 	 * @param _duration The total duration in seconds.
 	 * @param _feeSharing The address of fee sharing contract.
 	 * @param _vestingOwner The address of an owner of vesting contract.
+	 * @dev _vestingOwner should ALWAYS be multisig.
 	 * @return The four year vesting contract address.
 	 * */
 	function deployFourYearVesting(
```

Comment:
We will have to redeploy the factory should the logic change.
To avoid this we can pass the logic address as a param to the function or make the logic replaceable.

Reply:
Done. This is an excellent point.